hackathon_site/dashboard/frontend/src/api/api.ts

```diff
@@ -1,8 +1,11 @@
-import axios from "axios";
+import axios, { AxiosResponse } from "axios";
+
+// Re-export the response type, so it's available without needing to import axios
+export type { AxiosResponse } from "axios";
 
 export const SERVER_URL =
     process.env.NODE_ENV === "development"
-        ? process.env.REACT_APP_DEV_SERVER_URL.replace(/\/$/, "")
+        ? process.env.REACT_APP_DEV_SERVER_URL?.replace(/\/$/, "")
```

Comment:
would this set the variable to `undefined` if the env variable is undefined as well?

Reply:
Yes, good catch.

I want to avoid implicitly coding in a default `localhost:8000`, so I've updated it to check first and throw an error if it's not set (I'm not a huge fan of doing on `export let` now, but it's the cleanest way).